exercises/branching-conditionals/daysInMonthByNumber/daysInMonthByNumber.js

```diff
@@ -19,8 +19,11 @@ function daysInMonthByNumber(monthNum) {
   if (!Number.isInteger(monthNum) || monthNum < 1 || monthNum > 12) {
     throw new Error(`Expected a month number from 1-12, received: ${monthNum}`);
   }
+  else if (monthNum === 1 || monthNum === 3 || monthNum === 5 || monthNum === 7 || monthNum === 8 || monthNum ===10 || monthNum===12)
```

Comment:
Write out in English what you're trying to do here. If `monthNum` is `1`, what do you want to do?

Get it working for one input at a time. Once you have it working for 3-4 inputs, you'll see how to get it working for every input.